indico/util/string.py

```diff
@@ -615,7 +616,7 @@ def sanitize_css(self, style):
 def sanitize_html(string):
     css_sanitizer = IndicoCSSSanitizer(allowed_css_properties=BLEACH_ALLOWED_STYLES_HTML)
     return bleach.clean(string, tags=BLEACH_ALLOWED_TAGS_HTML, attributes=BLEACH_ALLOWED_ATTRIBUTES_HTML,
-                        css_sanitizer=css_sanitizer)
+                        protocols=BLEACH_ALLOWED_PROTOCOLS_HTML, css_sanitizer=css_sanitizer)
```

Comment:
That should not be applied globally, but only when sending emails that require it (ie make it a default-false arg of this function).

Reply:
I tried to define the 'sanitise_html' with default arg:
`app.add_template_filter(lambda s: Markup(sanitize_html(s or '', default_protocols=False)), 'sanitize_html')`
but when I use the filter
`{{ email_body | sanitize_html(True) }}`
got error:
"setup_jinja.<locals>.<lambda>() takes 1 positional argument but 2 were given"

Could you please help ?

Reply:
same error: "setup_jinja.<locals>.<lambda>() takes 1 positional argument but 2 were given"
when calling the filter with arg:
`app.add_template_filter(lambda s, **kwargs: Markup(sanitize_html(s or '', **kwargs)), 'sanitize_html')
def sanitize_html(string, *, allow_cid=False):
{{ email_body | sanitize_html(True) }}
`

Reply:
Can you please show the full diff for that file? It sounds like you added the argument somewhere it should not be (such as the `setup_jinja` function)